pkg/checks/external/ssl_util/main.go

```diff
@@ -0,0 +1,105 @@
+//Package ssl_util implements ssl check functions to kuberhealthy.
+//These test TLS connectivity to a host and check current expiration
+//status and time until certificate expiration.
+
+package ssl_util
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"net"
+	"strconv"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var TimeoutSeconds = 10
```

Comment:
I think `TimeoutSeconds` here should be lowercase because it does not need to be public / exported

Reply:
Leaving it public lets users tweak the timeout, which is probably okay.

Reply:
Left `TimeoutSeconds` as is per comment rom @integrii